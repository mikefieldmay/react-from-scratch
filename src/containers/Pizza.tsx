import React from "react";

import { PizzaImage } from "../components/PizzaImage/PizzaImage";

const Pizza: React.SFC = () => {
  return (
    <div>
      <h1>The Pizza</h1>
      <PizzaImage />
    </div>
  );
};

export default Pizza;
