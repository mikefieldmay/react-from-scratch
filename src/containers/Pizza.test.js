import React from "react";
import Pizza from "./Pizza";
import renderer from "react-test-renderer";

test("Link changes the class when hovered", () => {
  const component = renderer.create(<Pizza />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
