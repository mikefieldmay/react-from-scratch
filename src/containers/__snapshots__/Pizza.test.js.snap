// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div>
  <h1>
    The Pizza
  </h1>
  <div>
    <img
      src={
        Object {
          "process": [Function],
        }
      }
    />
  </div>
</div>
`;
